django-web-app/templates/index.html

```diff
@@ -61,15 +88,48 @@ <h1 class="font-mid-large font-primary">
             </p>
             <p>Creating a sense of false scarcity or pressure to act quickly, like countdown timers or
                 limited-time offers.</p>
-            <a href="{% url 'home:know-about-dp' %}" class="btn-secondary">Know More</a>
+            <a href="{% url 'home:knowDp' %}" class="btn-secondary">Know More</a>
 
         </div>
         <hr style="margin: 10px;" class="line-bottom">
     </section>
 </section>
+<div class="scrollToTop">
+    <div class="inner">
+        <i class="fa fa-arrow-up icon"></i>
+    </div>
+</div>
 {% endblock content %}
 
-
 {% block script %}
 <script src="{% static 'js/messages.js' %}"></script>
-{% endblock script %}
\ No newline at end of file
+<script>
+    // Select the scrollToTop element
+    var scrollToTop = document.querySelector(".scrollToTop");
+
+    // Function to calculate the scroll value and update the scrollToTop button
+    function calcScrollValue() {
+        let pos = document.body.scrollTop || document.documentElement.scrollTop;
+        let calcHeight = document.documentElement.scrollHeight - document.documentElement.clientHeight;
+        let percentval = Math.round((pos * 100) / calcHeight);
+
+        if (pos > 100) {
+            scrollToTop.style.display = "flex";
+        } else {
+            scrollToTop.style.display = "none";
+        }
+
```

Comment:
you have made changes in the index.html, 
button will only appear on home page, 
for it to appear on all pages, you should add it in base.html